app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_one :posts
+  has_many :comments, through: :posts
+  has_many :likes, through: :posts
+
```

Comment:
- [ ] According to the ERD diagram, the relationship between user and post is a `one-to-many relationship`. So change `has_one :posts` to `has_many :posts` and don't forget to add the foreign key `author_id` to all the relationships.